### PR TITLE
Send contentInfo information when updating HTML.

### PR DESCRIPTION
**Pull request by SergioEstevao**
## Description

Send content structure information to the native apps for them to be able to display it.

## How has this been tested?

This can be tested using the GB-mobile PR here: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2380

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/editor/src/components/provider/index.native.js [hunk 1]
```diff
@@ -242,6 +253,7 @@ export default compose( [
 			getEditedPostContent,
 			selectedBlockIndex: getBlockIndex( selectedBlockClientId ),
 			blockCount: getBlockCount( rootClientId ),
```
Comment (hypest) [inline packages/editor/src/components/provider/index.native.js:255]: Hey @SergioEstevao , I know that this line is not part of the PR but, now that we import `getGlobalBlockCount` what do you think about computing the block count using `getGlobalBlockCount()` [like here](https://github.com/WordPress/gutenberg/blob/73a79f177194d954805d80da2b4705c06a3e5f20/packages/editor/src/components/table-of-contents/panel.js#L80)?

**Review by hypest — APPROVED** [review]
LGTM @SergioEstevao ! I've only left one comment but I don't consider it blocking for the PR.

File: packages/editor/src/components/provider/index.native.js (hunk 1 above)
Comment (SergioEstevao) [reply packages/editor/src/components/provider/index.native.js]: Ok, I will update the code for that.

File: packages/editor/src/components/provider/index.native.js [hunk 2]
```diff
@@ -252,7 +252,7 @@ export default compose( [
 			title: getEditedPostAttribute( 'title' ),
 			getEditedPostContent,
 			selectedBlockIndex: getBlockIndex( selectedBlockClientId ),
-			blockCount: getBlockCount( rootClientId ),
+			blockCount: getGlobalBlockCount(),
```
Comment (hypest) [inline packages/editor/src/components/provider/index.native.js:265]: Looks like `getBlockCount` is not needed anymore and we can remove it from the extraction list above, right?
